internal/services/containers/kubernetes_cluster_other_resource_test.go

```diff
@@ -2606,3 +2606,38 @@ resource "azurerm_kubernetes_cluster" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, nodeOsUpgradeChannel)
 }
+
+func (KubernetesClusterResource) customCATrustCertificates(data acceptance.TestData, certsList []string) string {
```

Comment:
I'm not entirely sure why the test config is separated from the resource test - we should keep them in the same file so move the test definition into this file.